modules/rhpam-apb/roles/deploy-kieserver/tasks/main.yml

```diff
@@ -90,5 +103,6 @@
       k8s:
         state: '{{ state }}'
         definition: "{{ lookup('template', 'kieserver-dc.yml.j2') | from_yaml }}"
+      register: k8s_result
```

Comment:
is this registered fact used anywhere?

Reply:
It is probably some garbage left from my debugging.